### DISCUSSION TITLE
ContributionsGuide/document

### DISCUSSION THREAD

**Discussion by giscus** [Air-duino/document#5] [comment]
# ContributionsGuide/document

我们欢迎任何关于文档的新贡献！ 关于 AirMCU 有一个专用于文档的仓库，位于GitHub (https://github.com/Air-duino/document)中。该文档使用了VuePress (https://v2.vuepress.vuejs.org/)进行配置，并使用了VuePress Theme Hope (https://them...

https://arduino.luatos.com/ContributionsGuide/document.html

**Comment by HalfSweet** [comment]
评论测试
